app/views/further_education_payments/provider/_header.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= govuk_header(homepage_url: "https://www.gov.uk") %>
+
+<% nav_html = govuk_service_navigation(
+  navigation_items: provider_nav_items,
+  current_path: request.fullpath
+) %>
+
+<%
+  nav_document = Nokogiri::HTML.fragment(nav_html)
+  sign_out_link = nav_document.css("a").last
+  sign_out_link["data-method"] = "delete" if sign_out_link
```

Comment:
I think this is probably fine but we may want other consider linking to a page with a sign out button that can post (`_method=delete`) to the sessions#destroy end point to support signing out without JS. Probably one for a separate ticket

Reply:
agreed one for next time